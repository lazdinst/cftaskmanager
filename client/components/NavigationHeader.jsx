import React from 'react';
import { Container, Menu, Icon } from 'semantic-ui-react';

function NavigationHeader() {
  return (
    <div>
      <Menu fixed='top' inverted>
        <Container>
          <Menu.Item as='a' header>
            CF Task Management App
          </Menu.Item>
          <Menu.Item as='a' href='https://www.google.com/search?q=Talis+Lazdins'>Talis Lazdins</Menu.Item>
          <Menu.Item as='a' href='https://www.linkedin.com/in/talis-lazdins/'><Icon name='linkedin'/></Menu.Item>
          <Menu.Item as='a' href='https://github.com/lazdinst/'><Icon name='github'/></Menu.Item>
        </Container>
      </Menu>
    </div>
  );
}

export default NavigationHeader;