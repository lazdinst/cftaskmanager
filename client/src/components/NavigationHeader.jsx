import React from 'react';
import { Container, Menu, } from 'semantic-ui-react';

// Define the main app class
const NavigationHeader = () => {
  return (
    <div>
      <Menu fixed='top' inverted>
        <Container>
          <Menu.Item as='a' header>
            CF Task Management App
          </Menu.Item>
          <Menu.Item as='a'>Talis Lazdins</Menu.Item>
        </Container>
      </Menu>
    </div>
  );
}

export default NavigationHeader;