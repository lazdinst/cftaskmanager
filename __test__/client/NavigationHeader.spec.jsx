import React from 'react'
import { shallow, mount } from 'enzyme';
import NavigationHeader from '../../client/components/NavigationHeader.jsx'
import toJson from 'enzyme-to-json';

describe('<NavigationHeader />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NavigationHeader />);
  });

  it('should render', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
