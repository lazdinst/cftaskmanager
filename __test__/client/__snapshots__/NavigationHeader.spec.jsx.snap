// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationHeader /> should render 1`] = `
<div>
  <Menu
    fixed="top"
    inverted={true}
  >
    <Container>
      <MenuItem
        as="a"
        header={true}
      >
        CF Task Management App
      </MenuItem>
      <MenuItem
        as="a"
      >
        Talis Lazdins
      </MenuItem>
    </Container>
  </Menu>
</div>
`;
